test/cli/test_cassettes.py

```diff
@@ -153,7 +154,8 @@ async def test_replay(openapi_version, cli, schema_url, app, reset_app, cassette
 
 
 def test_multiple_cookies(base_url):
-    response = requests.get(f"{base_url}/success", cookies={"foo": "bar", "baz": "spam"})
+    headers = {"User-Agent": USER_AGENT}
```

Comment:
Could you please, add an explicit `assert` that the proper "User-Agent" header is present in `serialized["headers"]`?